packages/remix/src/build.ts

```diff
@@ -196,6 +197,157 @@ export const build: BuildV2 = async ({
       const rootDirectory = relative(repoRootPath, workPath);
       serverBuildPath = join(rootDirectory, serverBuildPath);
     }
+
+    if (remixConfig.images) {
```

Comment:
Is this a feature of `remix.config` or are you adding a new property that would only be used for Vercel?

If its the latter, then it should probably be `remixConfig.vercel.images` instead to avoid conflicts with Remix official configuration properties.

Reply:
Makes total sense. Done!